modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -387,7 +387,6 @@ up    <string name="player_sleep_in_one_chapter">Sleeping in 1 chapter</string>
     <!-- %1$d is the number of total chapters. %2$d is the number of hidden chapters. -->
     <string name="number_of_chapters_summary_plural">%1$d chapters &#x2022; %2$d hidden</string>
     <string name="number_of_chapters_summary_singular">1 chapter &#x2022; %d hidden</string>
-    <string name="select_one_chapter_message">Please select at least one chapter</string>
```

Comment:
No action is needed but it is sufficient to just delete the key from the main strings file, keys from other translations will be automatically deleted on next translation sync.